packages/dds/sequence-sequential-id/src/SharedStringWithSequentialIdFactory.ts

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import * as MergeTree from "@fluidframework/merge-tree";
+import { IChannelAttributes, IChannelServices,
+    IFluidDataStoreRuntime, IChannelFactory } from "@fluidframework/datastore-definitions";
+import { SharedString } from "@fluidframework/sequence";
+import { pkgVersion } from "./packageVersion";
+import { sharedStringWithSequentialIdMixin } from "./sharedStringWithSequentialId";
+
+export class SharedStringWithSequentialIdFactory implements IChannelFactory {
+    // TODO rename back to https://graph.microsoft.com/types/mergeTree/string once paparazzi is able to dynamically
+    // load code
+    public static Type = "https://graph.microsoft.com/types/mergeTree";
+
+    public static readonly Attributes: IChannelAttributes = {
+        type: SharedStringWithSequentialIdFactory.Type,
+        snapshotFormatVersion: "0.1",
+        packageVersion: pkgVersion,
+    };
+
+    public static segmentFromSpec(spec: any) {
+        const maybeText = MergeTree.TextSegment.fromJSONObject(spec);
+        if (maybeText) { return maybeText; }
+
+        const maybeMarker = MergeTree.Marker.fromJSONObject(spec);
+        if (maybeMarker) { return maybeMarker; }
+    }
+
+    public get type() {
+        return SharedStringWithSequentialIdFactory.Type;
+    }
+
+    public get attributes() {
+        return SharedStringWithSequentialIdFactory.Attributes;
+    }
+
+    /**
+     * {@inheritDoc @fluidframework/datastore-definitions#IChannelFactory.load}
+     */
+    public async load(
+        runtime: IFluidDataStoreRuntime,
+        id: string,
+        services: IChannelServices,
+        attributes: IChannelAttributes): Promise<SharedString> {
+        const sharedStringWithSequentialIdClass = sharedStringWithSequentialIdMixin();
```

Comment:
I'd remove "mixin" from the name, as that's implementation detail (i.e. it tells how function is implemented, not what function does - I think it should be the later).
How about createSharedStringWithSequentialIds() ?

Reply:
Or, it's a factory, so maybe createSharedStringWithSequentialIdsFactory? Too long :)
Not sure, naming is hard, but I'd appreciate if you spend a moment seeing if you can come up with better names here

Reply:
renamed